### PR TITLE
[RFR] Added e2e playbook

### PR DESCRIPTION
**Pull request by vatsalparekh**

### CODE REVIEW

File: tests/playbooks/e2e.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          kind: VirtualMachine
+          metadata:
+            creationTimestamp: null
+            labels:
+              kubevirt-vm: e2e-vm
+            name: e2e-vm
+          spec:
+            running: false
+            template:
+            metadata:
```
Comment (pkliczewski) [inline tests/playbooks/e2e.yaml]: We are missing indentation here .

File: tests/playbooks/e2e.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+              - name: disk0-pvc
+                persistentVolumeClaim:
+                claimName: e2e-pvc
+              - name: cloudinitvolume
+                cloudInitNoCloud:
+                userData: |-
+                  #cloud-config
+                  password: "redhat"
+                  chpasswd: { expire: False }
+    - name: Change VM status to running
```
Comment (pkliczewski) [inline tests/playbooks/e2e.yaml]: Shouldn't we wait for vm to be in `Running` state?

File: tests/playbooks/e2e.yaml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                    name: cloudinitdisk
+                    volumeName: cloudinitvolume
+                resources:
+                  requests:
+                    memory: 64Mi
+              terminationGracePeriodSeconds: 0
+              volumes:
+              - name: disk0-pvc
+                persistentVolumeClaim:
+                claimName: e2e-pvc
```
Comment (masayag) [inline tests/playbooks/e2e.yaml]: Could you extract the pvc_name to a variable and replace it where it is being used across this file ?

File: tests/playbooks/e2e.yaml (hunk 2 above)
Comment (vatsalparekh) [reply tests/playbooks/e2e.yaml]: @pkliczewski wait mechanism for VM?

Comment (pkliczewski) [reply tests/playbooks/e2e.yaml]: @vatsalparekh Yes, I think it could be useful to wait on vmi to become `Running`

Comment (pkliczewski) [reply tests/playbooks/e2e.yaml]: @vatsalparekh Let's add it in follow-up PR

**Comment by pkliczewski** [comment]
@masayag please review

**Pull request merged**